packages/terra-form/docs/ChoiceField.md

```diff
@@ -0,0 +1,22 @@
+# Choice Field
```

Comment:
What was the reasoning for naming this a ChoiceField, as opposed to something like RadioGroup?